opgincidentresponse/actions/keyword_handlers.py

```diff
@@ -1,12 +1,12 @@
 from response.core.models.incident import Incident
 from django.conf import settings
-
 from response.slack.models import CommsChannel
 from response.slack.decorators import keyword_handler
 
-import logging
-logger = logging.getLogger(__name__)
-
 @keyword_handler(['status page', 'statuspage'])
-def status_page_notification(comms_channel: CommsChannel, user: str, keyword: str, text: str, ts: str):
+def status_page_notification(comms_channel: CommsChannel):
     comms_channel.post_in_channel(f"ℹ️ You mentioned the Status Page - You can find our statuspage here: https://theofficeofthepublicguardian.statuspage.io/")
+
+@keyword_handler(['status page', 'statuspage'])
```

Comment:
Presumably this should hang off runbook keywords?
```suggestion
@keyword_handler(['run book', 'runbook'])
```

Reply:
Yeah realised once I was testing this locally just now 🤦  updated!